pkg/controller/controller_clusterservicebroker.go

```diff
@@ -39,7 +39,7 @@ const (
 	errorFetchingCatalogReason  string = "ErrorFetchingCatalog"
 	errorFetchingCatalogMessage string = "Error fetching catalog. "
 	errorSyncingCatalogReason   string = "ErrorSyncingCatalog"
-	errorSyncingCatalogMessage  string = "Error syncing catalog from ServiceBroker. "
+	errorSyncingCatalogMessage  string = "Error syncing catalog from ServiceBroker."
```

Comment:
Also I suspect we will have the same reasons/messages for namespaced service brokers, so we should probably prefix the variable names with `cluster-` too.